doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
+- Author(s): Emmanuel Baccelli  
```

Comment:
If the name with contact information is at the end anyways I see no purpose in having it here as well.